### PR TITLE
Add Colorful Clothing - Gloves - Distribution

### PR DESCRIPTION
**Pull request by Sabre142**
I intend to update this mod quite soon and so I didn't include any cleaning data for it.

### CODE REVIEW

**Comment by sibir-ine** [comment]
Should we wait until then to merge?

**Comment by Sabre142** [comment]
I will push out an update now and then provide its cleaning data.

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**
